src/settings/Settings.tsx

```diff
@@ -220,6 +221,26 @@ export const SETTINGS: { [setting: string]: ISetting } = {
             requiresRefresh: true,
         },
     },
+    [Features.NotificationSettings2]: {
```

Comment:
Could we perhaps use a better naming scheme here? Perhaps we could rename the old notif settings to legacy similarly to the `CallView`?

Reply:
I could do that for the actual classes, but I'm not sure what you're suggesting for renaming the labs flag?

Reply:
Perhaps `NotificationSettings2`?